src/System.Runtime.Extensions/src/System/IO/StringWriter.cs

```diff
@@ -178,8 +183,13 @@ public override void WriteLine(ReadOnlySpan<char> buffer)
             WriteLine();
         }
 
-        #region Task based Async APIs
+        public override void WriteLine(StringBuilder value)
+        {
```

Comment:
All of these overloads also need an "is disposed" check like:
https://github.com/MarcoRossignoli/corefx/blob/fd9d8612069863df29ccbf6000ce41f9b9241632/src/System.Runtime.Extensions/src/System/IO/StringWriter.cs#L89-L92

Reply:
ok

Reply:
done